index.html

```diff
@@ -61,7 +61,7 @@
 
   <section id="getting-started">
     <aside>
-    <h3>Alright. How do I get started?</h3>
```

Comment:
It is not a spelling error, it is an widely used alternative. In fact, we have used it in more than a few places in our blog. I appreciate the PR and typo fixes, but we're good with this one.